%
% LaTeX source of my resume
% =========================
%
% Heavily commented to to fit even LaTeX beginners (hopefully).
%
% See the `README.md` file for more info.
%
% This file is licensed under the CC-NC-ND Creative Commons license.
%


% Start a document with the here given default font size and paper size.
\documentclass[10pt,a4paper]{article}

% Set the page margins.
\usepackage[a4paper,margin=0.69in]{geometry}

% Setup the language.
\usepackage[english]{babel}
\hyphenation{Some-long-word}

% allow utf-8
\usepackage[utf8]{inputenc}

% Makes resume-specific commands available.
\usepackage{resume}
\usepackage{footmisc}

\renewcommand{\thefootnote}{\fnsymbol{footnote}}

\begin{document}  % begin the content of the document
\sloppy  % this to relax whitespacing in favour of straight margins


% title on top of the document
\maintitle{DAVID N.P. KOPPSTEIN}{}{}

\nobreakvspace{0.1em}  % add some page break averse vertical spacing

% \noindent prevents paragraph's first lines from indenting
% \mbox is used to obfuscate the email address
% \sbull is a spaced bullet
% \href well..
% \\ breaks the line into a new paragraph
\noindent\href{mailto:david.koppstein@gmail.coml}{david.koppstein\mbox{}@\mbox{}gmail.com}\sbull
\textsmaller{}+49 1707 740947 \sbull Skype: david.koppstein
\vspace{0.05in}

\spacedhrule{-0.2em}{-0.6em}

\roottitle{Education}

\headedsection
  {Massachusetts Institute of Technology}
  {\textsc{Cambridge, MA}} {%
  \headedsubsection
    {\textnormal{Ph.D, Biology}}
    {2015} {}
}

\headedsection
  {Yale University}
  {\textsc{New Haven, CT}} {%
  \headedsubsection
    {\textnormal{B.S., Molecular Biophysics and Biochemistry}}
    {2008} {}
}
\vspace{0.05in}

\spacedhrule{-0.2em}{-0.6em}

\roottitle{Experience}

\headedsection  % sets the header for the section and includes any subsections
  {Nikolaus Rajewsky Lab, Max Delbr√ºck Center for Molecular Medicine}
  {\textsc{Berlin, Germany}} {%
  \headedsubsection
    {Postdoctoral Researcher}
    {June \apo18 -- present}
    {\bodytext{Investigating the function of alternative polyadenylation during differentiation of neural lineages in single cells. Investigating coupling between polyadenylation and splicing using long reads.}}
}

\headedsection  % sets the header for the section and includes any subsections
  {Viral Immunology Systems Program, Kirby Institute, UNSW}
  {\textsc{Sydney, Australia}} {%
  \headedsubsection
    {Postdoctoral Researcher}
    {Apr \apo17 -- April \apo18}
    {\bodytext{Developed computational methods for analyzing single-cell transcriptomics data of immune cells. }}
}

\headedsection  % sets the header for the section and includes any subsections
  {Juno Therapeutics}
  {\textsc{Seattle, WA, USA}} {%
  \headedsubsection
    {Data Scientist}
    {Jan \apo16 -- Mar \apo17}
    {\bodytext{Analyzed and communicated immunosequencing analyses for internal projects and external academic collaborators. Conducted analyses for internal RNA-seq and vector integration profiling projects. Served as interim manager of the data science team. }}
}

\headedsection  % sets the header for a subsection and contains usually body text
  {AbVitro}
  {\textsc{Boston, MA, USA}} {%
  \headedsubsection
    {Data Scientist}
    {Mar \apo15 -- Dec \apo15}
    {\bodytext{Designed bioinformatics pipeline for single-cell immunosequencing. Mined receptor sequences from tumor-infiltrating lymphocytes and nominated candidates for screening. \textit{(Acquired by Juno Therapeutics).}}}
}

\headedsection  % sets the header for a subsection and contains usually body text
  {David Bartel Lab, MIT}
  {\textsc{Cambridge, MA, USA}} {%
  \headedsubsection
    {Graduate Student}
    {May \apo10 -- Mar \apo15}
    {\bodytext{Designed, implemented, and analyzed high-throughput sequencing experiments using the Illumina platform to investigate influenza's cap-snatching repertoire. Utilized 3P-seq to precisely examine alternative cleavage and polyadenylation in diverse genetic backgrounds and model organisms.}}
}

\headedsection
  {Mark Gerstein Lab, Yale University}
  {\textsc{New Haven, CT, USA}} {
  \headedsubsection
    {Bioinformatics Researcher}
    {June \apo08 -- June \apo09}
    {\bodytext{Designed and implemented bioinformatic methods to compare and calibrate RNA-seq and microarray data.}}
}

\headedsection  % sets the header for a subsection and contains usually body text
  {Joan Steitz Lab, Yale University}
  {\textsc{New Haven, CT, USA}} {%
  \headedsubsection
    {Student Researcher}
    {May \apo06 -- Apr \apo08}
    {\bodytext{Investigated the function of Herpesvirus saimiri Small U RNAs (HSURs).}}
}

\headedsection  % sets the header for a subsection and contains usually body text
  {Richard Ebright Lab, Rutgers University}
  {\textsc{Piscataway, New Jersey, USA}} {%
  \headedsubsection
    {Student Researcher}
    {May \apo04 -- Aug \apo04}
    {\bodytext{Mutagenized bacterial RNA polymerase and isolated mutants resistant to small molecules.}}
}

\spacedhrule{0.1em}{-0.4em}  % a horizontal line with some vertical spacing before and after

\roottitle{Teaching and Mentoring Experience}

\headedsection  % sets the header for a subsection and contains usually body text
  {Viral Immunology Systems Program, Kirby Institute, UNSW}
  {\textsc{Sydney, Australia}} {
  \headedsubsection
    {Student Supervisor}
    {Apr \apo17 -- Apr \apo18}
    {\bodytext{Cosupervised two Ph.D students, one Masters student, and one Honors student.
    Held weekly meetings with other supervisors and provided daily feedback and advice on single-cell sequencing projects.}}
}

\headedsection  % sets the header for a subsection and contains usually body text
  {Genetics, 7.03, MIT}
  {\textsc{Cambridge, MA, USA}} {%
  \headedsubsection
    {Teaching Assistant}
    {Feb \apo12 -- May \apo12}
    {\bodytext{Led review sections and held office hours. Contributed to exam creation and graded exams.}}
}

\headedsection  % sets the header for a subsection and contains usually body text
  {Quantitative Biology for Graduate Students, 7.57, MIT}
  {\textsc{Cambridge, MA, USA}} {
  \headedsubsection
    {Teaching Assistant}
    {Feb \apo10 -- May \apo10}
    {\bodytext{Responsible for computational lab component of course. Led computational labs, review sections,
    and held office hours. Created and graded exams and problem sets incorporating bioinformatic exercises.}}
}
\newpage

\roottitle{Publications and Preprints}  % a root section title
{\noindent Rizzetto S, \textbf{Koppstein D}, Samir J, Singh M, Reed JH, Cai CH, Lloyd AR, Eltahla AA, Goodnow CC, Luciani F. B-cell receptor reconstruction from single-cell RNA-seq with VDJPuzzle. 2018. \textit{Bioinformatics. doi:10.1093/bioinformatics/bty203}. \vspace{0.5em} \\
{\noindent Grigaityte K, Carter JA, Goldfless SJ, Jefferey EW, Hause RJ, Jiang Y, \textbf{Koppstein D}, Briggs AW, Church GM, Vigneault F, Atwal GS. Single-cell sequencing reveals $\alpha \beta$ chain pairing shapes the T cell repertoire. 2017. \textit{bioRxiv}.} \vspace{-0.8em} \\
{\noindent Briggs AW, Goldfless SJ, Timberlake S, Belmont BJ, Clouser CR, \textbf{Koppstein D}, Sok D, Vander Heiden JA, Tamminen MV, Kleinstein SH, Burton DR, Church GM, Vigneault F. 2017. Tumor-infiltrating immune repertoires captured by single-cell barcoding in emulsion. \textit{bioRxiv}. \vspace{0.5em} \\
{\noindent \textbf{Koppstein D}, Ashour J, and Bartel DP. Sequencing the cap-snatching repertoire of H1N1
influenza provides insight into the mechanism of viral transcription initiation. 2015. \textit{Nucleic Acids Research 43(10), 5052-5064}.} \vspace{0.5em} \\
{\noindent Hezroni H, \textbf{Koppstein D}, Schwartz M, Tabin CJ, Bartel DP, and Ulitsky I. 2015. Principles of long noncoding
RNA evolution derived from direct comparison of transcriptomes in 14 vertebrates. \textit{Cell Reports 11(7), 1110-1122}}. \vspace{0.5em} \\
{\noindent Nam J-W, Rissland OS, \textbf{Koppstein D}, Abreu-Goodger C, Jan CH, Agarwal V, Yildirim, Rodriguez A, and Bartel DP. 2014. Global Analyses of the Effect of Different Cellular Contexts on MicroRNA Targeting. \textit{Molecular Cell. 53, 1031-1043.}} \vspace{0.5em} \\
{\noindent Ulitsky I, Shkumatava A, Jan C, Subtelny AO, \textbf{Koppstein D}, Bell G, Sive H, and Bartel DP. 2012. Extensive alternative polyadenylation during zebrafish development. \textit{Genome Research. 22(10):2054-66.}} \vspace{0.5em} \\
{\noindent Agarwal A\footnote[1]{Authors contributed equally to this publication.}, \textbf{Koppstein D}\footnotemark[1], Rozowsky J, Sboner A, Habegger L, Hillier LW, Sasidharan R, Reinke V, Waterston RH, and Gerstein M. 2010. Comparison and calibration of transcriptome data from RNA-Seq and tiling arrays. \textit{BMC Genomics. 11:383.}} \vspace{0.3em} \\
{\noindent Mukhopadhyay J, Das K, Ismail S, \textbf{Koppstein D}, Jang M, Hudson B, Sarafianos S, Tuske S, Patel J, Jansen R, Irschik H, Arnold E, and Ebright RH. 2008. Myxopyronin, Corallopyronin, and Ripostatin Inhibit Transcription by Binding to the RNA Polymerase Switch Region. \textit{Cell. 135:295-307.}} \vspace{0.3em} \\

\spacedhrule{-0.4em}{-0.5em}  % a horizontal line with some vertical spacing before and after

\roottitle{Presentations and Posters}

{\noindent \textbf{Koppstein D}, Rizzetto S, Samir J, Singh M, Reed JH, Cai CH, Lloyd AR, Eltahla AA, Goodnow CC, Luciani F. VDJPuzzle: A computational method for BCR and TCR reconstruction from single-cell sequencing data. Australian Bioinformatics And Computational Biology Society. Oral presentation delivered in Adelaide, Australia, November 2017.} \vspace{-0.8em} \\

{\noindent \textbf{Koppstein D} Rizzetto S, Samir J, Singh M, Reed JH, Cai CH, Lloyd AR, Eltahla AA, Goodnow CC, Luciani F. VDJPuzzle: A computational method for BCR and TCR reconstruction from single-cell sequencing data. Australian Cellular Panomics Consortium. Poster presentation delivered in Melbourne, Australia, November 2017.} \vspace{-0.8em} \\

{\noindent \textbf{Koppstein D}, Rizzetto S, Samir J, Singh M, Reed JH, Cai CH, Lloyd AR, Eltahla AA, Goodnow CC, Luciani F. VDJPuzzle: A computational method for BCR and TCR reconstruction from single-cell sequencing data. Australian Society for Immunology. Oral presentation delivered in Bowral, Australia, November 2017.} \vspace{-0.8em} \\

{\noindent \textbf{Koppstein D}, Ashour J, Bartel D. Quantitative assessment of influenza's cap-snatching repertoire by RNA sequencing. RNA Society. Poster presentation delivered in Quebec City, Canada, June 2014.}

\spacedhrule{0.4em}{-0.6em}  % a horizontal line with some vertical spacing before and after

\roottitle{Fellowships and Awards}  % a root section

\begin{itemize}
\item Juno CEO Discretionary Award for Outstanding Performance, 2016
\item NSF Graduate Research Fellowship Program, Honorable Mention, 2011
\item Yale STARS II Research Fellowship, 2007-8
\end{itemize}

\spacedhrule{0.1em}{-0.5em}  % a horizontal line with some vertical spacing before and after

\roottitle{Activities}

\begin{itemize}
\item Whitehead Partner for High School Science Teacher Outreach, 2012
\item Organic Chemistry and Biochemistry Tutor, 2006-2010
\item Urban Cycling Skills and Safety Clinic Organizer, 2013-4
\item MIT Cycling Team Officer, 2012-2014
\end{itemize}

\spacedhrule{-0.2em}{-0.8em}  % a horizontal line with some vertical spacing before and after

\roottitle{References}  % a root section

{References are available upon request.}

\end{document}
